packages/shared/src/styles/responsive.module.scss

```diff
@@ -0,0 +1,103 @@
+/* -------------------------------------------------------------------------- */
+/* RESPONSIVE MIXINS                                                          */
+/* -------------------------------------------------------------------------- */
+
+// Variables ----------------------------------------------------------------
+$unit: 1px;
+$breakpoints: (
+    'mobile': 926,
+    'tablet': 1081,
+    'desktop': 1800,
+);
+
+// Functions ----------------------------------------------------------------
+@function unit($number) {
+    @return $number * $unit;
+}
+
+@function get-breakpoint($breakpoint, $type, $breakpoints: $breakpoints) {
+    $breakpoints-list: map-keys($breakpoints);
+    $new-breakpoint-index: '';
+    $new-breakpoint-value: '';
+    @if $type == 'prev' {
+        $new-breakpoint-index: (index($breakpoints-list, $breakpoint)) - 1;
+    } @else if $type == 'next' {
+        $new-breakpoint-index: (index($breakpoints-list, $breakpoint)) + 1;
+    }
+
+    @if $new-breakpoint-index != 0 {
+        $new-breakpoint-value: nth($breakpoints-list, $new-breakpoint-index);
+    }
+
+    @return $new-breakpoint-value;
+}
+
+// Mixins -------------------------------------------------------------------
+@mixin bp($breakpoint, $extend: '') {
```

Comment:
please add examples of how to use this mixin

Reply:
Example Added.